dev-libs/serdisplib/metadata.xml

```diff
@@ -1,12 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
-	<longdescription lang="en">
-		Library to drive displays with built-in controllers connected to
-		a parallel port (and serial port for i2c-displays).
-	</longdescription>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<upstream>
+		<bugs-to>https://sourceforge.net/p/serdisplib/bugs/</bugs-to>
 		<remote-id type="sourceforge">serdisplib</remote-id>
 	</upstream>
+	<longdescription>
+		Serdisplib started as a library to drive serial displays with built-in controllers.
+		Beginning with version 1.95 support was added for parallel driven displays.
+		Anyhow: the name 'serdisplib' will not change.
+		At the beginning it was only planned to add support for optrex 323 displays using
+		some patches to graphlcd, but soon i decided to create a separate library which can be used independently.
```

Comment:
This seems to be taken from some upstream readme, it does not really fit here. You could probably summarise the whole thing in 3 or 4 lines.

Reply:
Yes, thats the official description. I've changed it now.